src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1455,6 +1459,101 @@ static void triangles_command(std::string masterIP, int connFd, SQLiteDBInterfac
     }
 }
 
+static void streaming_triangles_command(std::string masterIP, int connFd, JobScheduler* jobScheduler,
+                                        bool *loop_exit_p, int numberOfPartitions) {
+    int uniqueId = JasmineGraphFrontEnd::getUid();
+    int result_wr = write(connFd, GRAPHID_SEND.c_str(), FRONTEND_COMMAND_LENGTH);
+    if (result_wr < 0) {
+        frontend_logger.error("Error writing to socket");
+        *loop_exit_p = true;
+        return;
+    }
+    result_wr = write(connFd, "\r\n", 2);
```

Comment:
Instead of this we should define a constant `CARRIAGE_RETURN_NEW_LINE` and refer it everywhere we get `write(connFd, "\r\n", 2);`

`write(connFd, CARRIAGE_RETURN_NEW_LINE.c_str(), CARRIAGE_RETURN_NEW_LINE.size());`

Reply:
Done

Reply:
Ok